src/main/java/io/wisetime/connector/api_client/ApiClient.java

```diff
@@ -45,13 +46,21 @@
   void tagDelete(String tagName) throws IOException;
 
   /**
-   * Add keywords to a tag. Existing keywords not be overwritten.
+   * Add keywords to a tag. Existing keywords will not be overwritten.
    *
    * @param tagName the tag to which to add the keywords
-   * @param addKeywordsRequest request contains list of keywords to be added
+   * @param additionalKeywords set of new keyword Strings to add to the tag
    * @throws IOException
    */
-  void tagAddKeywords(String tagName, AddKeywordsRequest addKeywordsRequest) throws IOException;
+  void tagAddKeywords(String tagName, Set<String> additionalKeywords) throws IOException;
```

Comment:
I've simplified the public API by removing unnecessary wrapping of the keywords into a request type.